automation/project/Dependencies.scala

```diff
@@ -5,6 +5,14 @@ object Dependencies {
   val akkaV = "2.4.17"
   val akkaHttpV = "10.0.5"
 
+  val workbenchModelV   = "0.8-d97f551"
+  val workbenchGoogleV  = "0.10-1b27c96-SNAP"
```

Comment:
I added a wb-libs depdendency to automation. The SNAP version refers to https://github.com/broadinstitute/workbench-libs/pull/47.

Reply:
FYI @vkumra-broad